zulipterminal/ui_tools/boxes.py

```diff
@@ -368,25 +369,35 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         if not reactions:
             return ''
         try:
-            reacts = defaultdict(int)  # type: Dict[str, int]
-            custom_reacts = defaultdict(int)  # type: Dict[str, int]
+            std_stats = Counter()  # type: typing.Counter[str]
+            custom_stats = Counter()  # type: typing.Counter[str]
```

Comment:
Could we use `std_reaction_stats` and `custom_reaction_stats` or something similar instead to increase verbosity?